packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -351,22 +349,17 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
             after: {
               0: {
                 target: 'flashFreshnessColors',
-                cond: 'hasFaceMatchedInOvalWithMinTime',
+                cond: 'hasFaceMatchedInOval',
                 actions: [
+                  'setFaceMatchTimeAndStartFace',
                   'updateEndFaceMatch',
                   'setupFlashFreshnessColors',
                   'cancelOvalMatchTimeout',
                   'cancelOvalDrawingTimeout',
                 ],
               },
-              0.1: {
-                target: 'ovalMatching',
-                cond: 'hasFaceMatchedInOval',
-                actions: 'setFaceMatchTimeAndStartFace',
-              },
               1: {
                 target: 'ovalMatching',
-                cond: 'hasNotFaceMatchedInOval',
               },
             },
```

Comment:
Since we are removing `0.1` step and there is no longer a condition inside `1` do we still need to use `after` here?

Reply:
You mean combine the two? So if `hasFaceMatchedInOval` do the lights and if `hasNotFaceMatchedInOval` we go back to `ovalMatching` (or is there a way to do like an `else`?)